src/components/ContextMenu/ContextMenu.stories.tsx

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import { Story } from "@storybook/react/types-6-0";
+import { Title, ArgsTable, Stories } from "@storybook/addon-docs";
+import ContextMenu, { ContextMenuProps } from "./ContextMenu";
+
+export default {
+  title: "Components/Navigation/ContextMenu",
+  component: ContextMenu,
+  parameters: {
+    docs: {
+      source: { type: "code" },
+      page: () => (
+        <>
+          <Title />
+          <ArgsTable of={ContextMenu} />
+          <Stories includePrimary title="Stories" />
+        </>
+      ),
+    },
+  },
+};
+
+const Template: Story<ContextMenuProps> = (args) => {
+  return <ContextMenu {...args} />;
+};
+
+export const Basic = Template.bind({});
+Basic.args = {
+  contents: [
+    {
+      text: "Apple",
+      onClick: () => {},
```

Comment:
action つけてもいいかも知れない。（リンゴの例だけど）

```suggestion
      onClick: action('clicked "Apple"'),
```